python/ray/workers/default_worker.py

```diff
@@ -124,6 +124,11 @@
     type=int,
     default=0,
     help="The PID of the process for setup worker runtime env.")
+parser.add_argument(
```

Comment:
What is this used for? It seems to only be here and in setup_runtime_env

Reply:
used for getting the uris, we can't retrieve the uris via job config because that would be the parent's job id

Reply:
I see.